scripts/update-pileup

```diff
@@ -13,7 +13,7 @@ npm run build
 # install pileup
 pwd
 echo "installing pileup.js ..."
-cd ../node_modules/pileup
-npm install ../../pileup.js
+cd ../node_modules
+npm install ../pileup.js
```

Comment:
We should just change these to absolute paths to remove the confusion. (we do this in bin/mango-submit , see here: https://github.com/bigdatagenomics/mango/blob/master/bin/mango-submit#L53